core/src/main/res/values/strings.xml

```diff
@@ -863,6 +865,13 @@
     <string name="notification_channel_auto_download">Auto Downloads</string>
     <string name="notification_channel_episode_auto_download">Shown when episodes have been automatically downloaded.</string>
 
+    <string name="notification_channel_new_episode">New Episode</string>
+    <string name="notification_channel_new_episode_description">Shown when a new episode of a podcast was found, where notifications are enabled</string>
+    <plurals name="new_episode_message">
+        <item quantity="one">%2$s has a new episode</item>
+        <item quantity="other">%2$s has %1$d new episodes</item>
+    </plurals>
```

Comment:
How about adding another string when multiple feeds have new episodes? Could be something like `%1$s and %2$s have %3$d new episodes`. This can't be done by Android automatically, switching between the messages would need to be done manually.

Reply:
ATM every feed gets one notification, so if there already is a notification for one feed you still get a sound if a new episodes drops on another feed (I don't think you get a sound when you update a notification). So I should have to change the notification code too.

Reply:
Okay, having one notification per subscription is fine, too, as long as you only get one notification sound during refresh, even if there are new episodes of multiple subscriptions. How about using notification groups, so that users can swipe away all of them at once if they want? https://developer.android.com/training/notify-user/group